specs/electra/beacon-chain.md

```diff
@@ -801,6 +806,54 @@ def slash_validator(state: BeaconState,
 
 ## Beacon chain state transition function
 
+### Execution engine
+
+#### Request data
+
+#### Engine APIs
+
+##### Modified `notify_new_payload`
+
+*Note*: The function `notify_new_payload` is modified to include the target number of blobs
+allowed per block.
+
+```python
+def notify_new_payload(self: ExecutionEngine,
+                       execution_payload: ExecutionPayload,
+                       parent_beacon_block_root: Root,
+                       target_blobs_per_block: uint64) -> bool:
```

Comment:
It should rather be a part of `is_valid_block_hash` and incorporated into a block hash validation process